hiro/GNUmakefile

```diff
@@ -74,19 +74,29 @@ ifeq ($(hiro.resource),)
   hiro.resource := $(hiro.path)/windows/hiro.rc
 endif
 
+ifneq ($(findstring windres,$(windres)),)
```

Comment:
You don't get (the LLVM version of) windres if you just install Clang by itself. Well, technically you do, because the same llvm-rc executable is capable of operating as a replacement for windres (the GCC resource compiler) or rc (the MSVC resource compiler) based on the name of the executable, but there's no way to override which one it masquerades as - it looks at argv[0] to decide.

Using llvm-rc as an rc replacement is actually preferable to windres in some ways, though. It outputs an architecture agnostic .res file instead of an .o file, and the latter just adds an extra wrinkle when cross compiling.